jupyterlab_automl/jupyterlab_automl/handlers.py

```diff
@@ -60,109 +95,141 @@ def create_automl_parent(client):
     return client.location_path(AuthProvider.get().project, "us-central1")
 
 
-def get_column_specs(client, tableSpecId, rowCount):
+def get_bucket_label(bucket):
+    if bucket.min == float("-inf"):
+        return "[" + str(bucket.min) + ", " + str(round(bucket.max)) + "]"
+    elif bucket.max == float("inf"):
+        return "[" + str(round(bucket.min)) + ", " + str(bucket.max) + "]"
+    else:
+        return "[" + str(round(bucket.min)) + ", " + str(round(bucket.max)) + "]"
+
+
+def get_detail_panel(column_spec, count):
+    chart_data = []
+    if column_spec.data_type.type_code == 3:
```

Comment:
Can we use enums here? I feel like having hardcoded integers everywhere could make things harder ro maintain in the future

Reply:
Agreed. Perhaps we should also have an enumeration for dataset type?